recipes-extended/wpe-framework/thunderhangrecovery/LICENSE

```diff
@@ -0,0 +1,223 @@
+
```

Comment:
Please remove this file.  There is already a LICENSE in the top level of this repository which is fine.